src/docs/asciidoc/reference/architecture/code-structure.adoc

```diff
@@ -1,2 +1,150 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+The purpose of this section is to explain the various elements and the interaction between them of a *ZetaPush application*.
```

Comment:
the various elements and the interaction between them of a ZetaPush application. -> the various elements of a ZetaPush application and the interaction between them.